Example2.js

```diff
@@ -0,0 +1,9 @@
+const baseValue = prompt('Enter the base of a triangle: ');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Kamala-Inc/testing-repo/pullRequest?prid=7028868)
```suggestion
const baseValue = prompt("Enter the base of a triangle: ");
```